assets/sass/_grid-settings.scss

```diff
@@ -1,108 +1,120 @@
-//
-// > ` @import 'vendor/neat/neat-helpers'; `
-//
-// > ` $desktop: new-breakpoint(max-width 1000px 12); `
-//
-// > ` $tablet: new-breakpoint(max-width 768px 8); `
-//
-// > ` $mobile: new-breakpoint(max-width 420px 4); `
-//
-// Style guide: Grid settings.2 Screen size
+/*
+Screen size
+
+Design for small screens first.
+
+Optimise for different screen sizes, but don’t make assumptions about what devices people are using.
+
+This framework uses 3 breakpoints:
+
+- Mobile: 4 columns, 420px maximum
+- Tablet: 8 columns, 768px maximum
+- Desktop: 12 columns, 1000px maximum
+
+Make sure you think about the number of columns used when resizing for small devices (eg see the [Bourbon Neat documentation](http://neat.bourbon.io/)). <!-- is this correct? -->
+
+> ` @import 'vendor/neat/neat-helpers'; `
+
+> ` $desktop: new-breakpoint(max-width 1000px 12); `
+
+> ` $tablet: new-breakpoint(max-width 768px 8); `
+
+> ` $mobile: new-breakpoint(max-width 420px 4); `
+
+Style guide: Grid settings.2 Screen size
+*/
 
 // Media Queries
 @import 'vendor/neat/neat-helpers';
 $desktop: new-breakpoint(max-width $max-width 12);
 $tablet: new-breakpoint(max-width 768px 8);
 $mobile: new-breakpoint(max-width 420px 4);
 
-// Debugging
-//
-// Turn on the visual grid to help debugging.
-//
-// ``` $visual-grid: true; ```
-//
-// Style guide: Grid settings.3 Debugging
+/*
+Debugging
+
+Turn on the visual grid to help debugging.
+
+``` $visual-grid: true; ```
+
+Style guide: Grid settings.3 Debugging
+*/
 
 // Toggle visibility of your grid columns for development/debugging.
 $visual-grid: false;
 
 // Make sure you set the viewport to the device-width in your head, eg
 // <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-// Attribution
-//
-// This guidance is in part adapted from [GOV.UK elements](http://govuk-elements.herokuapp.com/) under [Open Government Licence v3.0](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/), and [18F Draft US Web Design Standards](https://standards.usa.gov/getting-started/) under [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/legalcode).
-//
-// Style guide: Grid settings.4 Attribution
-
-// Last updated
-//
-// 23 June 2016 <!-- can we automate this please -->
-//
-// Style guide: Grid settings.5 Last updated
+/*
+Attribution
+
+This guidance is in part adapted from [GOV.UK elements](http://govuk-elements.herokuapp.com/) under [Open Government Licence v3.0](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/), and [18F Draft US Web Design Standards](https://standards.usa.gov/getting-started/) under [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/legalcode).
+
+Style guide: Grid settings.4 Attribution
+
+
+Last updated
+
+23 June 2016 <!-- can we automate this please -->
```

Comment:
(Not really related to the edits in this PR…)

I reckon we can remove these? What do you think @petronbot